packages/contracts-bedrock/scripts/FetchChainInfo.s.sol

```diff
@@ -0,0 +1,503 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Unit tests for this contract?

Reply:
Added in https://github.com/ethereum-optimism/optimism/pull/14930/commits/891d0c8aeac12f6025feef9d340754cfebefe5b2 and refined in subsequent commits